yt_dlp/extractor/afreecatv.py

```diff
@@ -253,6 +253,50 @@ def _real_extract(self, url):
         return self.playlist_result(entries, video_id, multi_video=True, **common_info)
 
 
+class AfreecaTVCatchStoryIE(AfreecaTVBaseIE):
+    IE_NAME = 'afreecatv:catchstory'
+    IE_DESC = 'afreecatv.com catch story'
+    _VALID_URL = r'https?://vod\.afreecatv\.com/player/(?P<id>\d+)/catchstory'
+    _TESTS = [{
+        'url': 'https://vod.afreecatv.com/player/103247/catchstory',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': '103247',
+        },
+        'playlist_count': 2,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        data = self._download_json(
+            'http://api.m.afreecatv.com/catchstory/a/view', video_id, headers={'Referer': url},
+            query={'aStoryListIdx': '', 'nStoryIdx': video_id}).get('data', [])
```

Comment:
see https://github.com/yt-dlp/yt-dlp/pull/10174#issuecomment-2195320898 about using `impersonate` instead of insecure requests

also, let's use `traverse_obj` below when we iterate over the data instead of `.get()` here
```suggestion
        data = self._download_json(
            'https://api.m.afreecatv.com/catchstory/a/view', video_id, headers={'Referer': url},
            query={'aStoryListIdx': '', 'nStoryIdx': video_id}, impersonate=True)
```